trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'decorator'
+
+module TrimmerDecoratorClass
+  include DecoratorClass
+  class TrimmerDecorator < Decorator
+    def correct_name
+      return @component.correct_name.slice(0, 9) if @component.correct_name.length > 10
+
+      @component.correct_name
+    end
```

Comment:
For the `TrimmerDecorator`, 

According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md), the method `correct_name` should trim longer words to a  maximum of 10 characters, not 9. 🤓

![tempsnip](https://github.com/ehmaddd/oop-school-library/assets/78251927/68703a05-d822-43ea-90be-4ae7cd7fb579)

Kindly fix that 👍

Reply:
Fixed as per instructions pl'